ssh.go

```diff
@@ -72,13 +72,16 @@ type ConnectionFailedCallback func(conn net.Conn, err error)
 type Window struct {
 	Width  int
 	Height int
+
+	WidthPixels  int
+	HeightPixels int
 }
 
 // Pty represents a PTY request and configuration.
 type Pty struct {
 	Term   string
 	Window Window
-	// HELP WANTED: terminal modes!
+	Modes  gossh.TerminalModes
```

Comment:
Docs? Especially since gossh.TerminalModes has no docs itself, surprisingly:

https://pkg.go.dev/golang.org/x/crypto/ssh#TerminalModes

Link to RFC and reference the "`VINTR`, `VQUIT`, etc" constants from `x/crypto/ssh`?

Reply:
Done